src/common/codeUtil.ts

```diff
@@ -3,7 +3,19 @@ export class CodeUtil {
      * trim unfoucs code.
      * @param origin any string
      */
-    public static purity(origin: string): string {
+    public static purity_greedy(origin: string): string {
+        if (!origin) return '';
+        // TODO: untest
+        return origin
+            .replace(/;.+/, '')
+            .replace(/".*?"/g, '')
+            .replace(/\{.*\}/g, '')
+            .replace(/ +/g, ' ')
+            .replace(/\bgui\b.*/gi, '')
+            .replace(/\b(msgbox)\b.+?%/gi, '$1');
+    }
+
+    public static purity(origin: string, greedy: boolean = false): string {
         if (!origin) return '';
         // TODO: untest
```

Comment:
If you'd like, I'd appreciate removing this comment, I don't know what it means.